website/content/partials/known-issues/1_16-default-lcq-pre-1_9-upgrade.mdx

```diff
@@ -0,0 +1,48 @@
+### Default lease count quota enabled when upgrading from Vault versions before 1.9
+
+#### Affected versions
+
+- 1.16+
+
+#### Issue
+
+Default lease count quotas are enabled in v1.16+ for new installs by consulting
+the version store.
+
+Before version 1.9, Vault did not maintain version history. As a result,
+upgrading from a version prior to 1.9 automatically enables a default lease
+count quota.
```

Comment:
```suggestion
Vault began tracking version history as of version 1.9. As of version 1.16, new
Vault installs automatically enable a lease count quota by consulting the
version history. If the version history does not include lease count quota
information, Vault assigns a default vault. As a result, when you upgrade Vault
from a version before 1.9, Vault automatically enables a default lease count quota.
```

Lets combine these two sections to make the connections more explicit for readers.

Reply:
The technical details are slightly different here, so I'm going to edit a bit, but I'll incorporate the feedback, thank you!